website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -1106,11 +1106,32 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
-		vector_info: {
+		build_info: {
 			description:       "Has a fixed value of 1.0. Contains build information such as rust and vector versions."
 			type:              "gauge"
 			default_namespace: "vector"
-			tags:              _internal_metrics_tags
+			tags:              _internal_metrics_tags & {
+				debug: {
+					description: "Level of debug info for Vector."
```

Comment:
This description can probably be improved, something like "Whether this is a debug build of Vector"

Reply:
Oh I didn't realize the debug flag was boolean till now.